submissions/Jigsaff/a-tiny-JS-world-post-OOP/index.js

```diff
@@ -77,8 +67,8 @@ class CatWoman extends Human {
     }
 }
 
-const dog = new Dog('Fred', 'male');
-const cat = new Cat('Luna', 'female');
+const dog = new Dog('Fred', 'male', 4);
+const cat = new Cat('Luna', 'female', 4);
 const woman = new Woman('Adele');
 const man = new Man('Arnold');
```

Comment:
Not all humans have 2 legs/hands. 
Also not all was fixed
[old comment](https://github.com/kottans/frontend-2021-homeworks/pull/354#discussion_r548998555)